norduser/service/fork.go

```diff
@@ -176,4 +182,15 @@ func (c *ChildProcessNorduser) StopAll() {
 			log.Println("failed to send a signal to norduser process: ", err)
 		}
 	}
+
+	doneChan := make(chan interface{})
+	go func() {
+		c.wg.Wait()
```

Comment:
Looks like wg.Wait() without wg.Add() ? I see only one wg.Add() in other place - but there it has corresponding wg.Wait(), here looks like it will wait forever.

Reply:
Ok, I got it: `It Blocks the execution until it’s internal counter becomes 0.` it will wait if there is something to wait